openquake/risklib/asset.py

```diff
@@ -140,10 +140,11 @@ def get_units(self, loss_types):
         for lt in loss_types:
             if lt.endswith('_ins'):
                 lt = lt[:-4]
-            if lt == 'area':
-                # NOTE: there's currently no trivial way to retrieve the
-                #       actual measurement unit from data (e.g. sqm or sqft)
-                unit = 'area'
+            if lt == 'area' and lt not in self.units:
+                # the unit might be missing in the exposure model xml
+                # NB: units are split by space afterwards, so we can't replace
```

Comment:
I HATE this. We must take a decision: or units are mandatory (so it is an error not to set them) or are useless so we must remove them. The middle ground a just a nonsensical maintenance burden.

Reply:
According to @nicolepaul, the <area> tag is often missing, so making it mandatory would imply needing to update all the grm exposure models. I suppose we should discuss it and make a decision.